components/calendar/test/calendar.test.js

```diff
@@ -92,6 +93,27 @@ describe('d2l-calendar', () => {
 			const expectedFocusDate = new Date(2015, 8, 2);
 			expect(calendar._focusDate).to.deep.equal(expectedFocusDate);
 		});
+
+		it('dispatches d2l-calendar-view-change event when user changes to previous month', async() => {
+			const calendar = await fixture(normalFixture);
+			const el = calendar.shadowRoot.querySelectorAll('d2l-button-icon')[0];
+			setTimeout(() => el.click());
```

Comment:
Suggest using the helpers as they actually perform the click with the mouse. You also don't need to run them in a `setTimeout` since they're async.

```suggestion
			clickElem(el);
```